### DISCUSSION TITLE
new exploit sucks

### DISCUSSION THREAD

**Discussion by Darcii** [3kh0/ext-remover#430] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
tbh theres no point in posting a exploit that requires something that any competent it department will disable i.e dev tools.

**Comment by rwg100** [comment, marked as answer]
сука блят rush b no stop!!!!

**Reply by Aka-but-nice** [reply]
> tbh theres no point in posting a exploit that requires something that any competent it department will disable i,e dev tools

That's not the new exploit, on the site click on New Point Blank, this does not need dev tools, it simply relies on the Same-Policy-Origin, the launcher was made by me and echo, but the idea was by Bypassi.
